crates/ln-dlc-node/src/ln/coordinator_event_handler.rs

```diff
@@ -354,10 +356,10 @@ where
                 .peer_manager
                 .get_peer_node_ids()
                 .iter()
-                .any(|(id, _)| *id == target_node_id)
+                .any(|(id, _)| *id == trader_id)
             {
                 tracing::info!(
-                    %target_node_id,
+                    %trader_id,
```

Comment:
:wrench: Prefer consistent naming i.e. `peer_id`.

Reply:
ok, changed it.